README.md

```
@@ -1,31 +1,89 @@
+
+```sh
+bun test             # one-shot
+bun test --watch     # watch mode
+bun run typecheck    # tsc --noEmit
+```
+
+## Architecture
 
-#### Mac errors & fixes in setup:
-`fatal error: 'openssl/ssl.h' file not found`.
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)